php_apcu/README.md

```diff
@@ -0,0 +1,80 @@
+# Agent Check: php_apcu
+
+## Overview
+
+This check monitors [php_apcu][1] through the Datadog Agent.
+
+## Setup
+
+Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][2] for guidance on applying these instructions.
+
+### Installation
+
+To install the php_apcu check on your host:
+
+
+1. Install the [developer toolkit]
+(https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
+ on any machine.
+
+2. Run `ddev release build php_apcu` to build the package.
+
+3. [Download the Datadog Agent](https://app.datadoghq.com/account/settings#agent).
+
+4. Upload the build artifact to any host with an Agent and
+ run `datadog-agent integration install -w
+ path/to/php_apcu/dist/<ARTIFACT_NAME>.whl`.
+
+#### APCu Setup
+
+Provide a URL to the collection program.
+
+APCu does not have a metric output by itself.
+
+* /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
+
+It will work if you refer to it as a php file, but you may want to configure it as follows
+
+```
+Alias /apcu-status /opt/datadog-agent/embedded/lib/python3.8/site-packages/datadog_checks/php_apcu/assets/exporter/apcu-dd-handler.php
+<Location /apcu-status>
+    Require all denied
+    Require local
+</Location>
+```
+
+### Configuration
+
+1. Edit the `php_apcu.d/conf.yaml` file, in the `conf.d/` folder at the root of your Agent's configuration directory to start collecting your php_apcu performance data. See the [sample php_apcu.d/conf.yaml][3] for all available configuration options.
+
+2. [Restart the Agent][4].
+
+### Validation
+
+[Run the Agent's status subcommand][5] and look for `php_apcu` under the Checks section.
+
+## Data Collected
+
+### Metrics
+
+See [metadata.csv][6] for a list of metrics provided by this check.
+
+### Service Checks
+
+php_apcu does not include any service checks.
+
+### Events
+
+php_apcu does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][7].
+
+[1]: **LINK_TO_INTEGRATION_SITE**
```

Comment:
Link to the php_apcu site

Reply:
fixed 28cdd6d7124fbadf91bb9b3e9f8ab47a1d1093df